configure.ac

```diff
@@ -934,10 +934,67 @@ if test "x$enable_backtrace" = xunwind-ptrace; then
       )],
       [AC_DEFINE([HAVE_LIBUNWIND_ELF_FILENAME], [1], [Define if libunwind has unw_get_elf_filename])
       AC_MSG_RESULT(yes)],
-      [AC_MSG_RESULT(no)]
+      [AC_MSG_RESULT([You have selected libunwind, but this version does not support retrieving file names])]
    )
 fi
 
+AC_ARG_ENABLE([demangling],
+              [AS_HELP_STRING([--enable-demangling],
+                              [enable demangling support for backtraces @<:@default=check@:>@])],
+              [],
+              [enable_demangling=no])
```

Comment:
Default value doesn't match what the help string says.

Reply:
I don't understand. Like `action-if-given` must be `@default=check@`?

Reply:
The fourth argument of `AC_ARG_ENABLE` specifies the default action if the user doesn't specify `--enable-demangling=whatever` when running `configure`. Here your default action is `enable_demangling=no`, which contradicts with the `@<:@default=check@:>@` in the help string.